src/main/kotlin/endpoint/RefundServerEndpoint.kt

```diff
@@ -26,16 +26,14 @@ class RefundServerEndpoint(
         .add(EthNotaryResponseMoshiAdapter())
         .add(BigInteger::class.java, BigIntegerMoshiAdapter())
         .build()!!
-    private val ethRefundAdapter = moshi.adapter(EthRefundRequest::class.java)!!
     private val ethNotaryAdapter = moshi.adapter(EthNotaryResponse::class.java)!!
 
     init {
         val server = embeddedServer(Netty, port = serverBundle.port) {
             routing {
-                get(serverBundle.ethRefund) {
-                    logger.info { "EthRefund invoked" }
-                    logger.info { "parameters:${call.parameters}" }
-                    call.respondText { onCallEthRefund(call.parameters["contract"]) }
+                get(serverBundle.ethRefund + "/{tx_hash}") {
```

Comment:
You can use $tx_hash

Reply:
It is a Ktor syntax to obtain parameter from URI.